CellProfiler_pipelines/README.md

```diff
@@ -0,0 +1,288 @@
+# CellProfiler Pipelines for NF1 Data Analysis
+
+In this module, I present my CellProfiler (CP) pipelines used for illumination correction, segmentation, and feature extraction.
+
+Since there are multiple methods are performing each of these steps, I decided to create two pipelines using [CellProfiler](https://cellprofiler.org/), version [4.2.4](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.2.4/index.html), a very commonly used and robust software for cell image analysis.
```

Comment:
```suggestion
Since there are multiple methods for performing each of these steps, I decided to create two pipelines using [CellProfiler](https://cellprofiler.org/), version [4.2.4](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.2.4/index.html), a very commonly used and robust software for cell image analysis.
```